### ISSUE TITLE
Confusing reference to invalid understanding doc from Understanding Success Criterion 2.4.7

### ISSUE THREAD

**Issue by mcking65** [w3c/wcag#3569]
The page [Understanding Success Criterion 2.4.7: Focus Visible | WAI | W3C](https://www.w3.org/WAI/WCAG22/Understanding/focus-visible.html) provides a confusing link to an understanding doc that is not valid -- [Understanding Success Criterion 2.4.11: Focus Appearance (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance-minimum). The link should be to [Understanding Success Criterion 2.4.13: Focus Appearance | WAI | W3C](https://www.w3.org/WAI/WCAG22/Understanding/focus-appearance.html).

The confusing link is the first link under the heading "Intent".

Related to issue #3328 for removing the invalid page.

**Comment by patrickhlauke** [comment]
Duplicate of https://github.com/w3c/wcag/issues/3555 - PR already filed https://github.com/w3c/wcag/pull/3557
